debian/bullseye/foreman-release/foreman-release.install

```diff
@@ -0,0 +1,2 @@
+debian/foreman.list                              etc/apt/sources.list.d/
+debian/theforeman-archive-keyring.asc            usr/share/keyrings/theforeman-archive-keyring.asc
```

Comment:
```suggestion
debian/theforeman-archive-keyring.asc            usr/share/keyrings/
```

the target is a directory, not the file name (the above creates `/usr/share/keyrings/theforeman-archive-keyring.asc/theforeman-archive-keyring.asc`)

Reply:
ah :\